extension/src/data/index.ts

```diff
@@ -9,8 +9,15 @@ import { uniqueValues } from '../util/array'
 import { DeferredDisposable } from '../class/deferred'
 import { isSameOrChild } from '../fileSystem'
 
+export type ExperimentsOutput = {
+  currentBranch: string
+  expShow: ExpShowOutput
+  order: { branch: string; sha: string }[]
```

Comment:
[F] Order is the key to duplication between branches. It is created from making multiple calls to `git log`.